_posts/2019/2019-12-17-what-working-with-a-distributed-team-taught-us-about-agile-team-collaboration.md

```diff
@@ -0,0 +1,80 @@
+It was the most convenient tool with great UX, a smooth support of your moderation flow and a variety of templates including nice team radar options. 
+When introducing Retrium, I also asked the team (even when collocated) to sit in front of their laptop with headsets on. 
+In this way, each team member participated individually and equally in the video call. 
+Sitting in a conference room was not an option anymore. 
+Even though it felt really awkward to sit in a room together while meeting in a video call, this was the most effective change to our discussion culture. 
+Suddenly everyone was equal and participation possibilities were equally distributed, too. 
+We could see all of the team members individually, could better get their mood by reading their face expressions and most importantly, I could see when someone unmuted, because she wanted to say something and react to that. 
+In the conference room setting, it was difficult to impossible for me to get those small signals. 
+At the same time, the team brought up the idea of a mini retrospective at the end of every meeting. 
+It took just five minutes and provided really valuable input for me. 
+The team members gave feedback on whether they like or disliked the new online tools we used, whether they wanted more or less strict moderation, and especially also gave feedback on how the sound quality was.
+
+## Improve your sound quality
+
+Everyone who works with distributed teams knows about this and I can already hear your sighs. 
+We had all the issues from bad microphone quality (so you could barely hear what your colleagues were saying) to Google Hangouts  automatically levelling the sound level of your microphones. 
+This was and still is my least favourite part about working in a distributed team, but the time spent was definitely worth it. 
+We kept looking for solutions until we had fixed all technical issues. 
+Annoying, but it worked. 
+I had even set up a meeting with my PO, who is working in a different location that I am, just to try out different devices for  45 minutes. 
+We also tried out things, that could affect sound quality (e.g. a laptop between you and the microphone), in order to make the teams sensible to what to watch out for. 
+It definitely helped to be in the remote position from time to time to experience the limiting factors yourself. 
+Another option is to record your meeting (e.g. the Standup) to become aware of sound quality issues and common patterns.
+
+## Remote first as a principle
+
+Early in our learning path towards a better remote collaboration, one of our developers recommend a podcast with Bryan Helming, Co-Founder of Zapir, a company that works exclusively remote. In this podcast, Bryan said, one of the company’s values they committed to was “Remote first”. 
+Like I had said in the beginning, examine every process on how you could make a physical process work for a distributed team.
```

Comment:
"had said" is past perfect but we need present perfect here. See https://www.quora.com/Which-is-correct-have-said-or-had-said

```suggestion
Like I have said in the beginning, examine every process on how you could make a physical process work for a distributed team.
```